indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/supervisor/SeekableStreamSupervisor.java

```diff
@@ -1260,6 +1260,30 @@ public void tryInit()
     }
   }
 
+  public boolean markSuperviosrIdleIfInactiveOnStartup()
```

Comment:
Can't the new logic just be a part of the existing method `checkIfStreamInactiveAndTurnSupervisorIdle()`?
That method is anyway invoked before creating new tasks and is already handling other required things.

Reply:
Since we are comparing offsets from metadata and also running this logic only on the very first run, I thought to separate this out to different method. Also, `checkIfStreamInactiveAndTurnSupervisorIdle` is invoked at the end of `runInternal` rather than the beginning.

Reply:
Just realized that order doesn't matter since we are creating tasks later. added the logic to existing method.